cms/envs/common.py

```diff
@@ -206,7 +206,14 @@
     # Turn off account locking if failed login attempts exceeds a limit
     'ENABLE_MAX_FAILED_LOGIN_ATTEMPTS': False,
 
-    # Allow editing of short description in course settings in cms
+    # .. toggle_name: FEATURES['EDITABLE_SHORT_DESCRIPTION']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: True
+    # .. toggle_description: This feature flag allows editing of short descriptions on the Schedule & Details page in
+    #   Open edX Studio. Set to False if you want to disable the editing of the course short description.
```

Comment:
Reading the linked PR, it seems like this setting only works when `ENABLE_MKTG_SITE` is set to False (the default). Do we need to add a warning about that?

Reply:
Good point, I think it makes sense to specify it for operators.

Reply:
@pdpinch and @BbrSofiane, I have checked the behavior for this toggle by setting `ENABLE_MKTG_SITE` to `True` and `False`, and in both cases `EDITABLE_SHORT_DESCRIPTION` works as expected. And in addition, I have checked the Django view and template which is responsible to render the Schedule & Details page but there is not any condition as you mentioned.